app/styles/_pipeline.less

```diff
@@ -0,0 +1,167 @@
+@build-pipeline-border-color: #d1d1d1;
+
+// Statuses
+@build-pipeline-new-color: #bee1f4;
+@build-pipeline-pending-color: #d1d1d1;
+@build-pipeline-in-progress-color: #0088ce;
+@build-pipeline-success-color: #3f9c35;
+@build-pipeline-failed-color: #cc0000;
+@build-pipeline-aborted-color: #ededed;
+
+.build-pipeline {
+  margin-bottom: 15px;
+  @media (min-width: 480px) {
+    margin-bottom: 0;
+  }
+  .build-summary, .stage {
+    .text-center();
+    min-width: 150px;
+  }
+  .build-summary, .stages-block {
+    border: 1px solid @build-pipeline-border-color;
+  }
+  .build-summary {
+    @media (min-width: 480px) {
+      border-right-width: 0;
+      flex-grow: 0;
+      flex-shrink: 0;
+      flex-basis: 200px;
+    }
+    min-height: 100px;
+    padding: 10px;
+    .status-icon {
+      margin-bottom: 3px;
+      .inverse {
+        font-size: 8px;
+        margin-bottom: 3px;
+      }
+      &.Complete .inverse {
```

Comment:
On Thu, May 19, 2016 at 1:12 PM, Sam Padgett <notifications@github.com>
wrote:

> In app/styles/_pipeline.less
> <https://github.com/openshift/origin-web-console/pull/20#discussion_r63917813>
> :
>
> > +  .build-summary {
> > +    @media (min-width: 480px) {
> > +      border-right-width: 0;
> > +      flex-grow: 0;
> > +      flex-shrink: 0;
> > +      flex-basis: 200px;
> > +    }
> > +    min-height: 100px;
> > +    padding: 10px;
> > +    .status-icon {
> > +      margin-bottom: 3px;
> > +      .inverse {
> > +        font-size: 8px;
> > +        margin-bottom: 3px;
> > +      }
> > +      &.Complete .inverse {
>
> I personally like it matching exactly. Otherwise you have to remember the
> transformation you made to the status in the view and make sure it matches
> in the CSS. More chances to introduce a bug.
>
> But I can change it if you disagree.
>

I only cared because it looks strange to me, its fine if you prefer it this
way.


> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-web-console/pull/20/files/b58a5686ba5fd1d681075d46d6f94b53d56a872b#r63917813>
>